io.github.prateekmedia.pstube

```diff
@@ -0,0 +1,40 @@
+app-id: io.github.prateekmedia.pstube
```

Comment:
this file is improperly named

Reply:
Whats the proper one then?

Reply:
Oh got it, I was just trying copy paste from previous one and missed it.